packages/api/src/clickhouse/index.ts

```diff
@@ -1019,6 +1024,71 @@ export const buildMetricSeriesQuery = async ({
     ],
   );
 
+  const quantile =
+    aggFn === AggFn.P50
+      ? '0.5'
+      : aggFn === AggFn.P90
+      ? '0.90'
+      : aggFn === AggFn.P95
+      ? '0.95'
+      : '0.99';
+
+  const histogramMetricSource = SqlString.format(
+    `
+      WITH points AS (
```

Comment:
I think we need one more CTE at the top here to properly calculate the group by request from the user, that aggregates bucket counts of all timeseries in a given group. (Ex. add up the counts for all 50 servers in one AWS region, if I wanted to group by AWS region)